thinc/util.py

```diff
@@ -176,17 +176,17 @@ def require_cpu() -> bool:  # pragma: no cover
 
 def prefer_gpu(gpu_id: int = 0) -> bool:  # pragma: no cover
     """Use GPU if it's available. Returns True if so, False otherwise."""
-    if not (has_cupy_gpu or has_torch_mps_gpu):
-        return False
-    else:
+    if has_gpu:
         require_gpu(gpu_id=gpu_id)
         return True
+    else:
+        return False
```

Comment:
```suggestion
    return has_gpu
```
(Feel free to ignore if you don't like it better - I don't feel strongly)